### DISCUSSION TITLE
Espressif DevCon22 - Arduino related talks

### DISCUSSION THREAD

**Discussion by VojtechBartoska** [espressif/arduino-esp32#7386] [Announcements]
Hello Dear contributors,

during this week, very first Espressif DevCon22 conference took place. 

The program included 2 talks related to this repository.

I'm sharing them here with you if you've missed them!

**1. Arduino Under the Hood - Lib Builder and Arduino as Component in ESP-IDF by @SuGlider (Arduino ESP32 Core Developer)** 
_[YT link](https://www.youtube.com/watch?v=8l29cTFS27w&t=30982)_

**2. News from Arduino: IDE 2.0 and Command Line Tools by Alessandro Ranellucci**
_[YT link](https://www.youtube.com/watch?v=BgRzpbZM5QY&t=31336)_

I hope those 2 talks can be interesting for you. Specially the first one, where you can find really practical info.

**Comment by SuGlider** [comment]
I fixed the links to point to the right place for repective presentations. Thanks!

**Reply by VojtechBartoska** [reply]
thanks!
